test/test_torch.py

```diff
@@ -8305,6 +8305,12 @@ def test_sizeof(self) -> None:
         self.assertEqual((sizeof_100 - sizeof_empty) // (sizeof_10 - sizeof_empty), 10)
         self.assertEqual((sizeof_100 - sizeof_empty) % (sizeof_10 - sizeof_empty), 0)
 
+    def test_resizable(self) -> None:
```

Comment:
This test might need to be decorated with @skipIfTorchDynamo() looks like it [fails on the dynamo shard](https://hud.pytorch.org/pr/119286)

Reply:
Reproduced this behavior with `@torch.compile` on local machine so `@skipIfTorchDynamo` should be the way to go.
For my info, when JIT compiled, does `x.numpy()` allocates another chunk of memory with the content of `x`?

Reply:
When JIT compiled, we actually transparently avoid the conversion to numpy entirely. So that's why this happens.

Reply:
Understood, thanks!